gpkit/interactive/sensitivity_map.py

```diff
@@ -119,8 +122,8 @@ def solution(self):
             return self.model.solution
 
     def _repr_latex_(self):
-        "iPython LaTeX representation."
-        return "$$\\require{color}\n"+self.latex+"\n$$"
+        "iPython LaTeX representation"
+        return "$$\\require{color}\n" +self.latex+"\n$$"+ self.color_list
```

Comment:
This change doesn't seem right to me. This adds a python list to a (latex) string? Has this been tested?